defaults/main.yml

```diff
@@ -35,6 +35,15 @@ __network_packages_default_wireless: ["{%
       if __network_wireless_connections_defined
       %}NetworkManager-wifi{% endif %}"]
 
+# check if any team connections are defined
+__network_team_connections_defined: "{{
+      ['team'] in network_connections|json_query('[*][type]') }}"
+
+# NetworkManager-team is required for team connections
+__network_packages_default_team: ["{%
```

Comment:
The variable also needs to be added to `__network_packages_default_nm`

Reply:
updated